kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -130,11 +131,18 @@
           facility: this.facility,
         };
         this.updateUser(this.userid, payload, this.role_new);
+        // if logged in admin updates role to learner, redirect to learn page
+        if (Number(this.userid) === this.session_user_id) {
+          if (this.role_new === constants.UserKinds.LEARNER.toLowerCase()) {
+            window.location.href = window.location.origin;
```

Comment:
We don't want to log out the user, but rather just re-direct them. We're not deleting the user, but just changing their role.